pkg/localstore/localstore_test.go

```diff
@@ -222,15 +223,15 @@ func TestGenerateTestRandomChunk(t *testing.T) {
 		t.Errorf("first chunk address length %v, want %v", addrLen, 32)
 	}
 	dataLen := len(c1.Data())
-	if dataLen != swarm.ChunkSize+swarm.SpanSize {
+	if dataLen != swarm.ChunkWithSpanSize {
```

Comment:
why is this change needed? these checks are all over the codebase, shouldn't we be PRing such changes to reflect everywhere?